src/System.IO.Pipelines/ReadableBufferReader.cs

```diff
@@ -41,11 +41,17 @@ public ReadableBufferReader(ReadCursor start, ReadCursor end) : this()
 
         public int Index => _overallIndex;
 
+        public Span<byte> Span => _currentMemory;
```

Comment:
Doesn't need to be exposed right now. I was playing with different patterns  earlier. I can remove it for now.